snapcraft/plugins/python2.py

```diff
@@ -48,6 +48,15 @@ def schema(cls):
         schema['properties']['requirements'] = {
             'type': 'string',
         }
+        schema['properties']['pip-packages'] = {
```

Comment:
Can you rename this to `python-packages`? `pip` is just the tool we use to reach the *python package index*